javaScript/script.js

```diff
@@ -1,185 +0,0 @@
-let nomeUsuario = { name: '' };
-let nomeDestinatario = "Todos";
-let mensagemArray = [];
-let usuariosOnline = [];
-let visibilidadeDaMensagem = "message";
-let checarVisibilidade;
-let focarMensagemNoChat;
-let ultimaMensagem = { time: 0 };
-let visivelParaQuem = "publicamente";
-let intevaloPegarMsg;
-
-enviarMsgTeclaEnter();
-
-function conferirNomeDigitado() {
-    nomeUsuario = {
-        name: document.querySelector("aside input").value
-    };
-
-    let promessaEnviada = axios.post('https://mock-api.driven.com.br/api/v4/uol/participants', nomeUsuario);
-    promessaEnviada.catch(checarErro);
-    promessaEnviada.then(liberarAcesso);
-}
-
-function checarErro(erro) {
-    if (erro.response.status === 400) {
-        alert("Esse nome já esta sendo utilizado, escolha outro por favor.")
-    }
-}
-
-function liberarAcesso() {
-    document.querySelector("aside").classList.add("none");
-    document.querySelector("body").classList.remove("overflow");
-
-    setInterval(manterOnline, 5000);
-    setInterval(pegarUsuariosOnline, 10000);
-    intevaloPegarMsg = setInterval(pegarMensagensDoServidor, 3000);
-
-    pegarUsuariosOnline();
-    atualizarReservada();
-}
-
-function manterOnline() {
-    axios.post('https://mock-api.driven.com.br/api/v4/uol/status', nomeUsuario)
-}
-
-function envioAutomatico(){
-    clearInterval(intevaloPegarMsg);
-    pegarMensagensDoServidor();
-    intevaloPegarMsg = setInterval(pegarMensagensDoServidor, 3000);
-}
-
-function pegarMensagensDoServidor() {
-    let promessaMensagens = axios.get('https://mock-api.driven.com.br/api/v4/uol/messages');
```

Comment:
Uma coisa legal de se fazer aqui, seria declarar uma constante global:
```
const url = 'https://mock-api.driven.com.br/api/v4/uol';
```
e sempre que for utilizá-la, adicionar o caminho correto:
```
const promessaMensagens = axios.get(`${url}/messages`);
```